src/store_io.cc

```diff
@@ -68,13 +68,13 @@ void
 storeClose(StoreIOState::Pointer sio, int how)
 {
     if (sio->flags.closing) {
-        debugs(20,3,HERE << "storeClose: flags.closing already set, bailing");
+        debugs(20,3,"storeClose: flags.closing already set, bailing");
```

Comment:
This PR automatically removes HERE. Nothing else should be required. Same for the other chunks in this file.

I think it would be possible to remove many (most?) duplicated prefixes like `storeClose:` but that should be done for _all_ debugs() lines, not just the ones with HERE. Otherwise, debugging output (and not just code) will be inconsistent and difficult to read. Do you want that "global" change (combined with the global parameter spacing change discussed earlier)?